services/tns_submission_queue/tns_submission_queue.py

```diff
@@ -862,6 +984,7 @@ def service(*args, **kwargs):
                     log(f"Error rolling back session: {str(e)}")
                 else:
                     try:
+                        traceback.print_exc()
```

Comment:
Useful?

Reply:
Yes! I want to see exactly where errors come from when we run this in prod. might delete later